Packs/Core/Integrations/CortexPlatformCore/CortexPlatformCore.py

```diff
@@ -740,6 +784,322 @@ def get_asset_group_ids_from_names(client: Client, group_names: list[str]) -> li
     return group_ids
 
 
+def get_appsec_rule_ids_from_names(client, rule_names: list[str]) -> list[str]:
+    """
+    Retrieves the IDs of AppSec rules based on their names using exact and partial matching.
+
+    Args:
+        client (Client): The client instance used to send the request.
+        rule_names (list[str]): List of AppSec rule names to retrieve IDs for.
+
+    Returns:
+        list[str]: List of AppSec rule IDs.
+
+    Raises:
+        DemistoException: If any rule names cannot be found in the system.
+    """
+    if not rule_names:
+        return []
+
+    fb = FilterBuilder()
+    fb.add_field("ruleName", FilterType.EQ, rule_names)
+    data = (
+        client.get_webapp_data(build_webapp_request_data(APPSEC_RULES_TABLE, fb.to_dict(), limit=200, sort_field="ruleName"))
+        .get("reply", {})
+        .get("DATA", [])
+        or []
+    )
+
+    lookup = {r["ruleName"].lower(): r["ruleId"] for r in data if r.get("ruleId")}
+    ids, found = [], set()
+
+    for name in rule_names:
+        n = name.lower()
+        rid = lookup.get(n) or next((v for k, v in lookup.items() if n in k), None)
+        if rid:
+            ids.append(rid)
+            found.add(name)
+
+    missing = set(rule_names) - found
+    if missing:
+        raise DemistoException(f"Missing AppSec rules: {', '.join(missing)}")
+
+    return ids
+
+
+def create_policy_command(client: Client, args: dict) -> CommandResults:
+    """
+    Creates a new policy in Cortex Platform with defined conditions, scope, and triggers.
+    Args:
+        client: The Cortex Platform client instance.
+        args: Dictionary containing policy configuration parameters including:
+            - policy_name: Required name for the new policy
+            - description: Optional policy description
+            - asset_group_names: Asset groups to apply the policy to
+            - conditions_*: Various condition parameters (finding type, severity, etc.)
+            - scope_*: Policy scope configuration parameters
+            - trigger_*: Policy trigger configuration (periodic, PR, CI/CD)
+
+    Returns:
+        CommandResults: Results object containing the created policy information with
+        readable output, outputs prefix, and raw response data.
+
+    Raises:
+        DemistoException: If policy name is missing or no triggers are enabled.
+    """
+    policy_name = args.get("policy_name")
+    if not policy_name:
+        raise DemistoException("Policy name is required.")
+
+    description = args.get("description", "")
+    asset_group_ids = create_policy_get_asset_groups(client, args)
```

Comment:
```suggestion
    group_names = argToList(args.get("asset_group_names"))
    asset_group_ids = get_asset_group_ids_from_names(client, group_names)
```
Why not like this instead of an extra wrapper function?